### ISSUE TITLE
Menu inaccessible

### ISSUE THREAD

**Issue by ZanryuLIVE** [rolandoislas/LightHost#37]
Repositioning my microphone today and attempted to access my VSTs through Light Host but the menu no longer appears when clicked.

Tried moving Light Host around, uninstalled, reinstalled, tried clearing appdata, and no changes, cannot find documentation. Any ideas?

**Comment by Toysrfun** [comment]
yeah it doesnt work and when it shows it disappears soon as you try to open it. very unstable i gave up looking for alternative

**Comment by drewjc75** [comment]
@ZanryuLIVE @Toysrfun Out of curiosity, are the two of you using display scaling? I have two 4k monitors and desktop scaling is set to 125%—and this made Light Host pop up the menu far away from the system tray icon. I was able to fix this by editing the properties of the shortcut in the start menu, selecting the checkbox for "Override high DPI scaling behavior" and choose "System (Enhanced)" in the dropdown. This made the menu appear where it was supposed to.

It's far from perfect and I'd love to see someone fork this project for more modern systems, but it does work this way.
